src/models/hit-types/item-hit.ts

```diff
@@ -0,0 +1,296 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Memoize } from 'typescript-memoize';
+import { BooleanField } from '../metadata-fields/field-types/boolean';
+import { DateField } from '../metadata-fields/field-types/date';
+import { MediaTypeField } from '../metadata-fields/field-types/mediatype';
+import { NumberField } from '../metadata-fields/field-types/number';
+import { StringField } from '../metadata-fields/field-types/string';
+
+/**
+ * A model that describes an item hit from the MDS API.
```

Comment:
MDS API?
do you mean /metadata/foo?

Reply:
I'll make these docs clearer that it's the metadata search backend (via the PPS) and not item metadata fetching.